pkg/lib/machine-change-auth.html

```diff
@@ -1,93 +1,55 @@
 <div class="modal-header">
-    <button type="button" class="close" data-dismiss="modal"
+    <button type="button" class="close"
         aria-hidden="true">&times;</button>
     <h4 class="modal-title" translate="yes">Log in to {{host}}</h4>
 </div>
 
 <div id="machine-change-auth" class="modal-body">
     {{^available}}
-    <p translate="yes">Cockpit was unable to log into {{#strong}}{{host}}{{/strong}}.</p>
-    {{/available}}
+    <p translate="yes">Cockpit was unable to log into
+    {{#strong}}{{full_address}}{{/strong}} and the host does not
+    accept password login or any of your SSH keys.</p>  {{/available}}
 
     {{#available}}
-    <p translate="yes">Cockpit was unable to log into {{#strong}}{{host}}{{/strong}}. You can change your authentication credentials below. {{#can_sync}}You may prefer to {{#sync_link}}synchronize accounts and passwords{{/sync_link}}.{{/can_sync}}</p>
+    <p translate="yes">Cockpit was unable to log into
+    {{#strong}}{{full_address}}{{/strong}} automatically. You might
+    need to provide your authentication credentials every time you
+    connect. You may want to set up your SSH keys for automatic
+    login.</p>  <br/>
+
+    <form class="ct-form">
+      <label translate="yes" class="control-label" id="login-authentication">Authentication</label>
+      <span class="radio radio-horizontal">
+        <label>
```

Comment:
Why is there a span here?

1. Spans are inline, this intended as a block
2. The span prevents ct-form from working with things below it (the label > input combo.

Reply:
I copied that from somewhere... :) Fixed.